include/date/date.h

```diff
@@ -81,7 +81,7 @@
 #ifdef _MSC_VER
 #   pragma warning(push)
 // warning C4127: conditional expression is constant
-#   pragma warning(disable : 4127)
+#   pragma warning(disable : 4127 4996)
```

Comment:
What is the code which causes 4996?

Reply:
I just copied this from the original UWP support vcpkg patch. Looks like it's originally from [here](https://github.com/microsoft/vcpkg/pull/8151#issuecomment-531175393). I don't have a Windows dev environment, so I can't test these changes directly, but I know it fixed the UWP CI tests in vcpkg.